From 3c7a37b555717c32605fed03dca96d19fe5f2d04 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
 <frederic.epitre@orange.fr>
Date: Fri, 19 Oct 2018 08:02:12 +0200
Subject: [PATCH] anaconda: skip NTP installation and setup in dom0

Dom0 has no direct network access, to this doesn't make sense anyway.

Fixes QubesOS/qubes-issues#2110
---
 pyanaconda/kickstart.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/kickstart.py b/pyanaconda/kickstart.py
index 1121e0928..5ac54b4fd 100644
--- a/pyanaconda/kickstart.py
+++ b/pyanaconda/kickstart.py
@@ -1715,6 +1715,9 @@ class Timezone(commands.timezone.F25_Timezone):
         self._disabled_chrony = False
 
     def setup(self, ksdata):
+        ### Skip the whole NTP setup in Qubes dom0
+        return
+
         # do not install and use NTP package
         if self.nontp or NTP_PACKAGE in ksdata.packages.excludedList:
             if iutil.service_running(NTP_SERVICE) and \
-- 
2.17.1

